src/ussd_app.js

```diff
@@ -230,16 +344,238 @@ go.app = function() {
                     " looking for?",
                 ].join("")),
                 choices: [
-                    new Choice("state_end", $("Nearest Clinic")),
-                    new Choice("state_end", $("MMC Clinic")),
-                    new Choice("state_end", $("HCT Clinic")),
+                    new Choice("nearest", $("Nearest Clinic")),
```

Comment:
https://github.com/praekelt/go-usaid-mmc/pull/40/files#diff-020cb63d1a6c4bb7c2c5da851dd51bb9R567 should have the following for that option
```
"search": {
                "msisdn": "+1234",
                "mmc": "true",
                "hct": "true"
},```